src/cli.rs

```diff
@@ -28,12 +28,18 @@ pub fn build_cli() -> App<'static, 'static> {
         )
         .arg(
             Arg::with_name("profile")
-                .short("p")
+                .short("p")// TODO: remove this to avoid confusion with setting project?
```

Comment:
Although it's a breaking change this is the time to do it.  Maybe -o or -r?  If it was called --config and -c that would be less confusing than having profiles and projects as concepts that have similar names, but that might be too aggressive. :)

Reply:
Thanks -- I like the suggestion, and it was easy to do. Furthermore, it cleans up the confusion between `--profile` and `config` commands.

Reply:
I commented on the change before seeing this discussion. The name "profile" was taken from AWS as was the notion of a multi-profile config file:

```
> aws --help
usage: aws [-h] [--profile PROFILE] [--debug]

optional arguments:
  -h, --help         show this help message and exit
  --profile PROFILE
  --debug
```

The name "config" implies pointing at a configuration file, I'd think. At least, that's how it works in most tools I use that take a `--config` option. If you want to re-appropriate the `-p` flag, I'd be cautious about inducing confusion. And I think there's value in having the tool parallel functionality and terminology already in use by other devops tools (particularly those that have wide exposure).

Reply:
Something else to consider... this may not just break for people. The default project name is currently named "default" (no upstream guarantee on that) and the default profile name is "default". For anyone using the short option (`-p`), things may just work for a while until a change is made and then things confusingly break.